packages/ui/src/composites/wui-chip/styles.ts

```diff
@@ -60,7 +75,8 @@ export default css`
     border: 1px solid var(--wui-color-accent-090);
   }
 
-  a[data-variant='shade'] > wui-image {
+  a[data-variant='shade'] > wui-image,
+  a[data-variant='shadeSmall'] > wui-image {
     border: 1px solid var(--wui-gray-glass-010);
```

Comment:
In most places, we use `box-shadow` instead of `border`. This allows us to keep the element sizes (width, height) same in the designs. (search "box-shadow: inset" in the repo). 

Maybe we need the same in this component too?